python/tvm/contrib/random.py

```diff
@@ -55,4 +55,29 @@ def uniform(low, high, size):
     return _api.extern(size, [], lambda ins, outs: _intrin.call_packed(
         "tvm.contrib.random.uniform", float(low), float(high), outs[0]), dtype='float32')
 
+
+def normal(mean, stddev, size):
```

Comment:
make API arguments consistent when there is numpy API available, 
https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.normal.html